app/bundles/EmailBundle/EventListener/LeadSubscriber.php

```diff
@@ -102,13 +103,16 @@ protected function addEmailEvents(LeadTimelineEvent $event, $state)
                 } else {
                     $eventName = $label;
                 }
-
+                if ('failed' == $state) {
+                    $state = 'sent';
```

Comment:
I think we should indicate in the UI that it was failed and not sent (in some cases failed = failure to send at all)